pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -284,6 +467,46 @@ func checkUpdate(spec *wildflyv1alpha1.WildFlyServerSpec, statefuleSet *appsv1.S
 	return update
 }
 
+// listing pods which belongs to the WildFly server
+//   the pods are differentiated based on the selectors
+func (r *ReconcileWildFlyServer) getPodsForWildFly(w *wildflyv1alpha1.WildFlyServer) (*corev1.PodList, error) {
+	podList := &corev1.PodList{}
+	labelsForWildfly := labelsForWildFly(w)
+	delete(labelsForWildfly, labelWildflyOperatorInService)
```

Comment:
would it be simpler to put the labelWildflyOperatorInService when the Pod is actually active instead of deleting it when it should not be queried?

I envisioned `labelsForWildFly`as "static" labels that did not depend on the state of the cluster but it'd not be true anymore.

Reply:
yeap, ok, +1

Reply:
I changed this.